controllers/disruption_controller_test.go

```diff
@@ -178,7 +168,7 @@ var _ = Describe("Disruption Controller", func() {
 	AfterEach(func() {
 		// delete disruption resource
 		_ = k8sClient.Delete(context.Background(), disruption)
-		Eventually(func() error { return expectChaosPod(disruption, 0) }, timeout).Should(Succeed())
+		Eventually(func() error { return expectChaosPod(disruption, 0) }, timeout*2).Should(Succeed())
```

Comment:
Is this `*2` on timeout expected?

Reply:
Yes, I've seen the `afterEach` (and only that) not finish deleting within a single reconcile call, and then it pauses 9 seconds before trying again. This ends up failing the next test because of leaked pods. Not sure why this PR would be slower on delete